presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -302,6 +302,18 @@ public void createPartitionedTable(Session session, HiveStorageFormat storageFor
 
         assertUpdate(session, "INSERT INTO test_partitioned_table " + select, 1);
         assertQuery(session, "SELECT * from test_partitioned_table", select);
+        assertQuery(session,
+                "SELECT * from test_partitioned_table WHERE" +
+                        " 'foo' =  _partition_string" +
```

Comment:
fixed. Made the spacing after cast calls in this select statement consistent, but didn't touch the rest of the class.